VaccinationSystem/VaccinationSystem/Data/ApplicationDbContext.cs

```diff
@@ -14,9 +14,9 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
         protected override void OnModelCreating(ModelBuilder builder)
         {
             base.OnModelCreating(builder);
```

Comment:
Po tym megre'u zgubiłaś linijkę w której baza jest seedowana z Twojej konfiguracji

Reply:
nie bardzo rozumiem, co jeszcze powinnam dodać?

Reply:
public DbSet<Disease> Diseases { get; set; }
i to samo dla Vaccines, Vaccinations itp